packages/passport-interface/src/StampPCDUtils.ts

```diff
@@ -0,0 +1,30 @@
+import { sha256 } from "js-sha256";
+import { ProveRequest } from "./RequestTypes";
+
+export function hashRequest(req: ProveRequest): string {
+  const reqString = JSON.stringify(req);
+  return sha256(reqString);
+}
+
+export interface PendingStampPCD {
+  status: StampPCDStatus;
+
+  /**
+   * The type of PCD that a server is producing a stamp for.
+   */
+  pcdType: string;
+
+  /**
+   * A hash of the ProveRequest using hashRequest. Stored to avoid
+   * people re-sending the same request many times and clogging
+   * the proving queue.
+   */
+  hash: string;
+}
+
+export enum StampPCDStatus {
+  IN_QUEUE = "in queue",
```

Comment:
this is a personal pref but i generally avoid spaces in enum string values: `in_queue`